datahub-web-react/src/app/shared/tags/EditDescriptionModal.tsx

```diff
@@ -0,0 +1,71 @@
+import { Typography, Modal, Button, Form } from 'antd';
+import React, { useState } from 'react';
+import styled from 'styled-components';
+import MDEditor from '@uiw/react-md-editor';
+
+const MarkDownHelpLink = styled(Typography.Link)`
+    position: absolute;
+    right: 0;
+    top: -18px;
+    font-size: 12px;
+`;
+
+type Props = {
+    title: string;
+    description?: string | undefined;
+    onClose: () => void;
+    onSubmit: (description: string) => void;
+    isAddDesc?: boolean;
+};
+
+export default function EditDescriptionModal({ title, description, onClose, onSubmit, isAddDesc }: Props) {
+    const [updatedDescription, setUpdatedDescription] = useState(description || '');
+
+    return (
+        <Modal
+            title={title}
+            visible
+            width={900}
+            onCancel={onClose}
+            okText={isAddDesc ? 'Submit' : 'Update'}
+            footer={
+                <>
+                    <Button onClick={onClose}>Cancel</Button>
+                    <Button onClick={() => onSubmit(updatedDescription)} disabled={updatedDescription === description}>
```

Comment:
Please add analytics for this like in `AddOwnerModal.tsx`

Reply:
Fixed and Committed.

[https://github.com/linkedin/datahub/pull/4132/commits/f314d27106cd3135e7def6ac9db381a764eb4d73](url)